platform/source/mbed_retarget.cpp

```diff
@@ -819,6 +983,7 @@ long __lseek(int fh, long offset, int whence)
 int _lseek(FILEHANDLE fh, int offset, int whence)
 #endif
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Can you remove _lseek entirely?

Reply:
The content removed is not just for `_lseek`, it is also the content of the `_sys_seek` function. If any of the system I/O functions are redefined, then they must all be redefined. So I cannot remove `_lseek`.